src/test/java/game/InMemoryGameEntityRepoTest.java

```diff
@@ -0,0 +1,41 @@
+package game;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class InMemoryGameEntityRepoTest {
+    GameEntityRepository gameRepo;
+    Map<Integer, GameEntity> inMemoryDb;
+    GameEntity gameEntity;
+
+    @BeforeEach
+    void setUp() {
+        inMemoryDb = new HashMap<Integer, GameEntity>() {
+        };
+        gameEntity = new GameEntity(1, 2, 3);
+        inMemoryDb.put(gameEntity.returnGameId(), gameEntity);
+        gameRepo = new InMemoryGameEntityRepo(inMemoryDb);
+    }
+
+    @Test
+    void getEntity() {
+        assertEquals(gameRepo.getEntity(gameEntity.returnGameId()), gameEntity);
+    }
+
+    @Test
+    void save() {
```

Comment:
https://www.c-sharpcorner.com/UploadFile/dacca2/fundamental-of-unit-testing-understand-aaa-in-unit-testing/

Reply:
šitą testą pagal AAA reikia perdaryt. Pavadinimą irgi patikslink tuo klausimu kas kada ar kaip turėtų būt vykdoma